Tests/FluentAssertions.Specs/Execution/ContextDataSpecs.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using FluentAssertions.Execution;
+using Xunit;
+using Xunit.Sdk;
+
+namespace FluentAssertions.Specs.Execution
+{
+    public class ContextDataSpecs
```

Comment:
🤔 Aren't they still `AssertionScope` tests and thus should be named accordingly?

Reply:
Yeah, I was not sure on this, I had it as `AssertionScope.ContextDataSpecs.cs` and `AssertionScopeContextDataSpecs.cs` at some point. But as `AssertionScopeSpecs` is really about live cycle, and I felt this was really about a specific aspect not related to that.
Any suggestion what you would like?

Reply:
I prefer `AssertionScope.ContextDataSpecs.cs`.

Reply:
I'll take it the same applies for `ChainingApiSpecs.cs` and `MessageFormattingSpecs.cs`?